tools/UpdateVendors/Program.cs

```diff
@@ -100,18 +99,28 @@ private static void RewriteCsFileWithStandardTransform(string filePath, string o
 
         private static void UpdateVendor(
             string libraryName,
-            string branchDownload,
+            string downloadUrl,
             string[] pathToSrc,
             Action<string> transform = null)
         {
             Console.WriteLine($"Starting {libraryName} upgrade.");
 
             var zipLocation = Path.Combine(DownloadDirectory, $"{libraryName}.zip");
             var extractLocation = Path.Combine(DownloadDirectory, $"{libraryName}");
+            var vendorFinalPath = Path.Combine(_vendorProjectDirectory, libraryName);
+            var sourceUrlLocation = Path.Combine(vendorFinalPath, "_source_url.txt");
```

Comment:
Consider renaming `_source_url.txt` to something like `_last_downloaded_source_url.txt` to make it clear the this file holds the last downloaded version. At first glance I thought we now had to put the url in that file instead of hard-coding it in C# code.

Reply:
Resolved in latest.

Reply:
Thanks!